### ISSUE TITLE
I can't find "BasicRequestHelper" anywhere

### ISSUE THREAD

**Issue by geoffrothman** [mindbody/API-Examples#18]
I've tried this in python 2.7.x and 3.X and it tells me that its not defined. This is imported in many of these files. any ideas?

**Comment by vacoj** [comment]
That is a file in the repository.  See:  https://github.com/mindbody/API-Examples/blob/master/Python/BasicRequestHelper.py

Also, there is an example of it being used here:
https://github.com/vacoj/MINDBODY-Django/blob/master/helpers/mbosoap/BasicRequestHelper.py

Let me know if you hit any more snags after looking at those.  I'm happy to help.

**Comment by geoffrothman** [comment]
Thanks so much @vacoj ! I've got everything working now except for that bloody message about the Engine not being configured correctly in the settings file for Django. I did a manage.py diffsettings and here's what it says. Can I use a dummy database with this or do I for sure need to use Postgres?

DATABASES = {'default': {'USER': '', 'ENGINE': 'django.db.backends.dummy', 'OPTIONS': {}, 'AUTOCOMMIT': True, 'TEST': {'NAME': None, 'MIRROR': None, 'CHARSET': None, 'COLL
